packages/cashier/src/stores/payment-agent-transfer-store.ts

```diff
@@ -1,54 +1,71 @@
 import { action, computed, observable } from 'mobx';
+import { PaymentAgentDetailsResponse, PaymentAgentTransferResponse } from '@deriv/api-types';
 import { routes } from '@deriv/shared';
 import Constants from 'Constants/constants';
 import ErrorStore from './error-store';
+import {
+    TExtendedPaymentAgentListResponse,
+    TPaymentAgent,
+    TPaymentAgentTransferReceipt,
+    TPaymentAgentTransferConfirm,
+    TPaymentAgentTransferRequest,
+    TRootStore,
+    TTransferLimit,
+    TWebSocket,
+} from 'Types';
 
 export default class PaymentAgentTransferStore {
-    constructor({ WS, root_store }) {
+    constructor(public WS: TWebSocket, public root_store: TRootStore) {
         this.root_store = root_store;
         this.WS = WS;
     }
 
     @observable container = Constants.containers.payment_agent_transfer;
-    @observable error = new ErrorStore();
+    @observable error: TRootStore['modules']['cashier']['error'] = new ErrorStore();
     @observable is_payment_agent = false;
     @observable is_try_transfer_successful = false;
     @observable is_transfer_successful = false;
-    @observable confirm = {};
-    @observable receipt = {};
-    @observable transfer_limit = {};
+    @observable confirm: TPaymentAgentTransferConfirm = {};
+    @observable receipt: TPaymentAgentTransferReceipt = {};
+    @observable transfer_limit: TTransferLimit = {};
+    @observable onRemount: () => void = null;
```

Comment:
add `onRemount` variable

Reply:
@george-usynin-binary You also need to create an action for assigning amount to it.

Reply:
hey @Bahar-FS, guess that you wanted to type `onRemount` instead of `amount`
will add this action in both stores
```
 @action.bound
    setOnRemount(func: VoidFunction): void {
        this.onRemount = func;
    }
```